src/goto-symex/README.md

```diff
@@ -150,6 +186,10 @@ incremented (from 1 to 2) and the L0 counter is reset (back to 2, after
 having been incremented up to 4). The L0 counter then increases every
 time we access x as we walk through the function.
 
+\subsection L0 Level 0 versioning
+
+TODO
```

Comment:
This is for threads.

Reply:
yes this only  valid when threads are present, will add comment.